package.json

```diff
@@ -20,7 +20,13 @@
     "./assets/fill/*.svg": "./assets/fill/*.svg",
     "./assets/light/*.svg": "./assets/light/*.svg",
     "./assets/regular/*.svg": "./assets/regular/*.svg",
-    "./assets/thin/*.svg": "./assets/thin/*.svg"
+    "./assets/thin/*.svg": "./assets/thin/*.svg",
+    "./bold/*.svg": "./assets/bold/*.svg",
+    "./duotone/*.svg": "./assets/duotone/*.svg",
+    "./fill/*.svg": "./assets/fill/*.svg",
+    "./light/*.svg": "./assets/light/*.svg",
+    "./regular/*.svg": "./assets/regular/*.svg",
+    "./thin/*.svg": "./assets/thin/*.svg"
```

Comment:
I like this, aliasing makes it feel much nicer 👍

Reply:
yes, I prefer it a lot! I think whenever 2.0.0 hits then removing `/assets` exports is warranted, but staying true to semver is the way to go for now